setup.py

```diff
@@ -89,7 +89,7 @@
       author_email='',
       url='',
       keywords='web fedora pyramid',
-      packages=find_packages(),
+      packages=find_packages(exclude=['tests']),
```

Comment:
Thanks @jarifibrahim!

This doesn't seem to be working. You can test by running ```python setup.py install``` - you'll see that ```/usr/lib/python2.7/site-packages/bodhi-2.1.9-py2.7.egg/bodhi/tests``` is getting installed. I think this will work though:

```
packages=find_packages(exclude=('bodhi.tests', 'bodhi.tests.*')),
```

Let me know when you want me to take another look, and thanks for the contribution!